docs/guidelines/Composer-help-contents.md

```diff
@@ -0,0 +1,148 @@
+# Bot Framework Composer Help Content Guidelines
+
+Help contents refer to any texts that appear throughout Composer user interface (UI) that will help users to use the tool. Clear, concise, and comprehensible help contents will improve users' overall learning experience and achieve what they set out to build.
+
+In this article, we drafted some guidelines on how to write the help contents in Composer UI, including button names, menu names, instructional copy, tooltips, error messages, watermark texts, and help links pointing to [Composer official documentation](https://docs.microsoft.com/composer/). These guidelines are based on [Microsoft guidelines on linking to articles from the user interface](https://review.docs.microsoft.com/help/contribute/contribute-link-to-articles-from-the-user-interface?branch=master), and general observation of [Azure Bot Service](https://azure.microsoft.com/services/bot-service/) and [Power Virtual Agent](https://powervirtualagents.microsoft.com/) User Interfaces.
+
+This document includes:
+
+- [Bot Framework Composer Help Content Guidelines](#bot-framework-composer-help-content-guidelines)
+- [Terminology](#terminology)
+- [Guidelines](#guidelines)
+  - [General guidelines](#general-guidelines)
+  - [Button names](#button-names)
+  - [Menu names](#menu-names)
+  - [Instructional copy](#instructional-copy)
+  - [Tooltips](#tooltips)
+  - [Error messages](#error-messages)
+  - [Watermark texts](#watermark-texts)
+  - [Help links](#help-links)
+- [Resources](#resources)
+
+# Terminology
```

Comment:
What do you think about also mention the distinction between instructional copy, tooltip, and watermark here too? For example:

**Instructional copy**: instructional text that provides users with information about what to do.
* **Tooltip**: texts displayed in an informational text box when hovering over an item. Tooltips are intended for short definitions of a particular UX element.
* **Watermark text**: text that lives inside an edit box or entry field. Watermark text is meant to give an example input.
* **Help link**: context-sensitive help (CSH) link added to the user interface to help users with a concept or procedure while using a product.

Reply:
My understanding:
* Instructional copy: instructional, tells users what do to.
* tooltip: informational, inform users what it is.
* watermark text: more instructional than informational, usually with an example.

I can refine the terminology of "watermark text".